client/valor/schemas/symbolic/types.py

```diff
@@ -233,9 +233,7 @@ def __validate__(cls, value: typing.Any):
         NotImplementedError
             This function is not implemented in the base class.
         """
-        raise NotImplementedError(
-            f"Variable of type `{cls.__name__}` cannot be assigned a value."
-        )
+        pass
```

Comment:
can we just delete this method rather than passing?

fix docstring if not

Reply:
No, its a placeholder for subclasses. Ill try and find the convention for specifying this

Reply:
Added to the docstring saying that its a placeholder.